internal/types/kruizePayload/updateResult.go

```diff
@@ -36,7 +37,10 @@ type ErrorData struct {
 
 func GetUpdateResultPayload(experiment_name string, containers []map[string]interface{}) []UpdateResult {
 	payload := []UpdateResult{}
-	df := dataframe.LoadMaps(containers)
+	df := dataframe.LoadMaps(
+		containers,
+		dataframe.WithTypes(types.CSVColumnMapping),
+	)
```

Comment:
Do we need this df creation in two places?

Reply:
Yes, because `containers` value which it passed in this function is a map.